classes/abstracts/ActionScheduler_Abstract_ListTable.php

```diff
@@ -527,32 +593,49 @@ protected function maybe_render_actions( $row, $column_name ) {
 		return $actions;
 	}
 
+	/**
+	 * Process the bulk actions.
+	 *
+	 * @return void
+	 */
 	protected function process_row_actions() {
 		$parameters = array( 'row_action', 'row_id', 'nonce' );
 		foreach ( $parameters as $parameter ) {
-			if ( empty( $_REQUEST[ $parameter ] ) ) {
+			if ( ! isset( $_REQUEST[ $parameter ] ) || empty( $_REQUEST[ $parameter ] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
 				return;
 			}
 		}
 
-		$method = 'row_action_' . $_REQUEST['row_action'];
+		$action = sanitize_text_field( wp_unslash( $_REQUEST['row_action'] ) ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended, WordPress.Security.ValidatedSanitizedInput.InputNotValidated
+		$row_id = sanitize_text_field( wp_unslash( $_REQUEST['row_id'] ) ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended, WordPress.Security.ValidatedSanitizedInput.InputNotValidated
+		$nonce  = sanitize_text_field( wp_unslash( $_REQUEST['nonce'] ) ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended, WordPress.Security.ValidatedSanitizedInput.InputNotValidated
+		$method = 'row_action_' . $action; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
 
-		if ( $_REQUEST['nonce'] === wp_create_nonce( $_REQUEST[ 'row_action' ] . '::' . $_REQUEST[ 'row_id' ] ) && method_exists( $this, $method ) ) {
-			$this->$method( $_REQUEST['row_id'] );
+		if ( wp_create_nonce( $action . '::' . $row_id ) === $nonce && method_exists( $this, $method ) ) {
```

Comment:
I realize you didn't introduce this and are just making a change to satisfy the yoda rule, but thinking about it this isn't the right way to check a nonce.

*Optional* thing we can slip in:

```suggestion
		if ( wp_verify_nonce( $nonce, $action . '::' . $row_id ) && method_exists( $this, $method ) ) {
```